core/src/main/scala/org/bitcoins/core/psbt/PSBT.scala

```diff
@@ -857,7 +848,11 @@ object InputPSBTRecord {
     override val value: ByteVector = porCommitment
   }
 
-  case class Unknown(key: ByteVector, value: ByteVector) extends InputPSBTRecord
+  case class Unknown(key: ByteVector, value: ByteVector)
+      extends InputPSBTRecord {
+    require(key.size > 1)
+    require(PSBTInputKeyId.fromKey(key) == PSBTInputKeyId.UnknownKeyId)
```

Comment:
same as above, also, do you need both requires? Does the second not cover the first?